### PR TITLE
Implement mar-today tests

### LINKED ISSUE #835: Implement mar-today tests coverage
## Task
To support issue #806 
we will need top implement mar-today tests 
because  of their complexity, they need to be broken down into a new issue.

## Details
- step 1
- step 2

### PR DESCRIPTION
**Pull request by viniciusarre**
Closes #835

Works towards #806

### CODE REVIEW

File: client/src/inside/components/mar/tests/mar-today.spec.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      expectedScheduleTimes.map(time => {
+        const filter = result.filter(sch => sch.hour24 === time)
+        filter.length.should.be.greaterThan(0)
+      })
+    })
+  })
+  
+  it('isTimeValid', () => {
+    isTimeValid('08:00').should.equal(true)
+    isTimeValid('0800').should.equal(false)
```
Comment (bryan-gilbert) [inline client/src/inside/components/mar/tests/mar-today.spec.js:58]: Separate the valid from the invalid into two tests. Add more invalid tests.

File: client/src/inside/components/mar/tests/mar-today.spec.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+describe('mar-today tests', () => {
+  it('getSchedule', () => {
+    should.doesNotThrow(() => {
+      const result = getSchedule(medOrders)
+      marRecords = result.map(r => r.marRecord)
+      should.exist(result)
+      expectedScheduleTimes.map(time => {
+        const filter = result.filter(sch => sch.hour24 === time)
+        filter.length.should.be.greaterThan(0)
+      })
```
Comment (bryan-gilbert) [inline client/src/inside/components/mar/tests/mar-today.spec.js:52]: Compact test for the positive case.
Need tests for the negative cases.
